pages/home/home.html

```diff
@@ -0,0 +1,116 @@
+  <link rel="stylesheet" href="./home.css">
+  <title>eCommerce | Home</title>
+</head>
+<body>
+  <div class="body-center">
+    
+    <!-- NAVBAR  -->
+
+    <nav class='navbar'>
+      <div class="logo">
+        <h2>BlackT</h2>
+      </div>
+          
+    
+
+      <form class="form">
+        <input type='text' class="form-input small-text" placeholder="Enter Product">
+      </form> 
+                                                                                                                      
+      <div class="right-nav">
+        <button class="btn primary-btn">Login</button>
+        <ul>                                                    
+          <li>
+            <div class="badge">
+              <i class="fas fa-heart fa-2x"></i>
+              <div class="badge-number">1</div>
+            </div>
+          </li>
+          <li>
+            <div class="badge">
+              <i class="fas fa-shopping-cart fa-2x"></i>
+              <div class="badge-number">1</div>
+            </div>
+          </li>                                 
+        </ul>
+      </div>
+      <button class="navbar-btn">
+        <span class="hamburger"></span>
+      </button>
+    </nav>
+    <!-- 5 column layout -->
+    
+
+
+    <div class="grid grid-column-6">
+      <div class="grid-aside">
+        <img class='responsive-img' src="https://images.unsplash.com/photo-1643133277936-9f93d8792522?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHx0b3BpYy1mZWVkfDMyfHhqUFI0aGxrQkdBfHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60">
+      </div>
+      <div class="grid-main-content">
+        <img class='responsive-img' src="https://images.unsplash.com/photo-1643133277936-9f93d8792522?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHx0b3BpYy1mZWVkfDMyfHhqUFI0aGxrQkdBfHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60">
+      </div>
+      <div class="grid-aside">
+        <img class='responsive-img' src="https://images.unsplash.com/photo-1643133277936-9f93d8792522?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHx0b3BpYy1mZWVkfDMyfHhqUFI0aGxrQkdBfHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60">
+      </div>
+      <div class="grid-aside">
+        <img class='responsive-img' src="https://images.unsplash.com/photo-1643133277936-9f93d8792522?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHx0b3BpYy1mZWVkfDMyfHhqUFI0aGxrQkdBfHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60">
+      </div>
+      <div class="grid-aside">
+        <img class='responsive-img' src="https://images.unsplash.com/photo-1643133277936-9f93d8792522?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHx0b3BpYy1mZWVkfDMyfHhqUFI0aGxrQkdBfHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60">
+      </div>
+      <div class="grid-aside">
+        <img class='responsive-img' src="https://images.unsplash.com/photo-1643133277936-9f93d8792522?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHx0b3BpYy1mZWVkfDMyfHhqUFI0aGxrQkdBfHxlbnwwfHx8fA%3D%3D&auto=format&fit=crop&w=500&q=60">
+      </div>
+    </div>
```

Comment:
On screens, they have given text on the image. I guess that's a requirement.
![image](https://user-images.githubusercontent.com/62175113/152561285-816f61cb-2cfc-4037-bdcf-33243cbee189.png)

Reply:
> On screens, they have given text on the image. I guess that's a requirement. ![image](https://user-images.githubusercontent.com/62175113/152561285-816f61cb-2cfc-4037-bdcf-33243cbee189.png)

Done